python/llm/setup.py

```diff
@@ -344,6 +343,7 @@ def setup_package():
                         "npu": npu_requires,
                         "xpu-2-1": xpu_21_requires,
                         "xpu-lnl": xpu_lnl_requires,
+                        "xpu-arl": xpu_lnl_requires,
```

Comment:
Currently `xpu-lnl` and `xpu-arl` have the same requirements. @jason-dai Please let us know for any comments for the options :) Thank you